python/cuml/cuml/internals/base.pyx

```diff
@@ -792,3 +792,25 @@ class UniversalBase(Base):
         """
 
         return False
+
+    def _check_cpu_model(self):
```

Comment:
Can we delete this method from `ProxyEstimator` now that it is implemented here?